Makefile

```
@@ -0,0 +1,19 @@
+all: Makefile.coq
+	$(MAKE) -f $< $@
+
+install: Makefile.coq
+	$(MAKE) -f $< $@
+
+uninstall: Makefile.coq
+	$(MAKE) -f $< $@
+
+clean:
+	if [ -e Makefile.coq ]; then $(MAKE) -f Makefile.coq $@; fi
+	$(RM) Makefile.coq* .*.aux
+
+test:
+	$(MAKE) -C tests clean
+	$(MAKE) -C tests
+
+Makefile.coq: _CoqProject
+	coq_makefile -f $< -o $@
```

```suggestion
	$(COQBIN)coq_makefile -f $< -o $@
```
Please use the version of Coq the user requests, not just the one in PATH